boards/esp32-h2-devkitm-1.json

```diff
@@ -0,0 +1,29 @@
+{
+  "build": {
+    "core": "esp32",
+    "f_cpu": "96000000L",
```

Comment:
Is not fully correct. Missing an entry. Should be:
```
    "core": "esp32",
    "f_cpu": "96000000L",
    "f_flash": "64000000L",
    "f_image": "48000000L",
    "flash_mode": "qio",
    "mcu": "esp32h2",
    "variant": "esp32h2"
```

Reply:
Thanks @Jason2866 ! I'll push these changes shortly. Anything else I should update or add to get this approved?

Reply:
I am not related to Platformio. A user like you. Providing now and than a PR ;-)